lightkurve/lightcurve.py

```diff
@@ -14,13 +17,17 @@
 
 from astropy.stats import sigma_clip
 from astropy.table import Table
+from astropy.io import fits
+from astropy.time import Time
 
 from .utils import running_mean
-
+from . import PACKAGEDIR
 
 __all__ = ['LightCurve', 'KeplerLightCurve', 'TessLightCurve',
            'iterative_box_period_search']
 
+log = logging.getLogger(__name__)
```

Comment:
Should we declare ``log = logging.getLogger(__name__)`` on every module or should we import the one from ``__init__``? @christinahedges @barentsen

Reply:
The nice thing about putting `log = logging.getLogger(__name__)` at the top of every module is that it allows you to control the stream handler and verbosity at the module-level.  It also allows you to print the origin of the log message (e.g. `lightkurve.targetpixelfile`).

Reply:
awesome!